docs/source/index.rst

```diff
@@ -129,8 +129,8 @@ System on Module Designs
    :widths: 50 50
    :header-rows: 1
 
-   * - OAK-SOM
-     - OAK-SOM-Pro
+   * - OAK-SoM
```

Comment:
I’m not sure if we should do SoM or SOM here.   Let’s see what Bryan Norman thinks (not sure if he has a github yet).